core/src/main/scala/quasar/datasource/kafka/Config.scala

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2020 Precog Data
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package quasar.datasource.kafka
+
+import slamdata.Predef._
+
+import argonaut.Argonaut._
+import argonaut._
+import cats.data.NonEmptyList
+import cats.syntax.either._
+import quasar.connector.DataFormat
+
+case class Config(
+  bootstrapServers: NonEmptyList[String],
+  groupId: String,
+  topics: NonEmptyList[String],
+  decoder: Decoder,
+  format: DataFormat
+) extends Product with Serializable {
+  def isTopic(topic: String): Boolean = topics.exists(_ == topic)
+
+  // No sensitive information at this time
+  def sanitize: Config = this
+  def reconfigure(patch: Config): Either[Config, Config] = patch.asRight
+}
+
+object Config {
+
+  implicit def nelEJ[A: EncodeJson]: EncodeJson[NonEmptyList[A]] = _.toList.asJson
```

Comment:
Can we use the `EncodeJson` that [argonaut provides](https://github.com/argonaut-io/argonaut/blob/a31a53c97b52b84bde61861559e28ae013f8a130/argonaut-cats/shared/src/main/scala/argonaut/EncodeJsonCats.scala#L25)?